tomviz/LoadDataReaction.cxx

```diff
@@ -168,16 +168,16 @@ DataSource* LoadDataReaction::loadData(const QStringList& fileNames,
   bool child = options["child"].toBool(false);
 
   DataSource* dataSource(nullptr);
-  QString fileName;
+  QString firstFile;
```

Comment:
This feels odd, not sure if it is necessary but if you and @cjh1 want it for additional clarity it is fine by me.